sda-commons-server-kms-aws/build.gradle

```diff
@@ -0,0 +1,19 @@
+dependencies {
+  compile project(':sda-commons-starter')
```

Comment:
Modules shouldn't depend on the starter module but instead only on the direct dependencies that are actually need.

Reply:
I think, `sda-commons-server-dropwizard` is the dependency you are looking for.